internal/admin/rbac.go

```
@@ -0,0 +1,96 @@
+
+func (h *handler) editorMaxDailyCostCents() int64 {
+	if h.deps.YAMLConfig == nil {
+		return 0
+	}
+	return h.deps.YAMLConfig.Features.AdminDashboard.EditorLimits.MaxDailyCostLimitCents
+}
+
+func (h *handler) validateEditorCostLimit(r *http.Request, cents int64) error {
+	role, err := h.auth.userRole(r)
+	if err != nil {
+		return nil
```

golangci-lint (2.12.2)

[error] error is not nil (line 84) but it returns nil

(nilerr)